tests/Book/BookProviderTest.php

```diff
@@ -64,7 +64,17 @@ public function testGetCreditsUsesCorrectToolServerURL() {
 	}
 
 	private function apiWithResponse( $status, $header, $body ) {
-		$responses = [ new Response( $status, $header, $body ) ];
+		// Set up all responses that are used by the above tests. Most don't need to return any data.
+		$responses = [
+			new Response( $status, $header, $body ),
```

Comment:
I'm a bit confused as to why you need all these Response objects here. May I ask why is this necessary and just having the first line is not enough for the current tests?

Reply:
umm... no idea! hehe

I've removed them. Everything seems fine. :-)

Reply:
Oh, well: https://github.com/wsexport/tool/pull/272/checks?check_run_id=1422501542#step:7:189

I guess I tried the same thing last week!

Reply:
Sorry, I'm not explaining it well! :) It's because in a clean install we have to run the Refresh process, and that takes another few requests. We don't care what the return, with the exception of the namespaces one. I've added a comment.

To replicate the failure, try deleting the cache files in your `var/` and running this test.